### PR TITLE
Fix broken links and references in CONTRIBUTING

### PR DESCRIPTION
**Pull request by mohanraj-r**
* Fix broken link to stack overflow
* Fix broken references to file and test that don't exist anymore

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 Before raising an issue, make sure you have checked the open and closed issues to see if an answer is provided there.
-There may also be an answer to your question on [stackoverflow](stackoverflow.com/questions/tagged/pact).
+There may also be an answer to your question on [stackoverflow](https://stackoverflow.com/questions/tagged/pact).
 
 Please provide the following information with your issue to enable us to respond as quickly as possible.
 
 1. The relevant versions of the packages you are using.
 1. The steps to recreate your issue.
-1. An executable code example where possible. You can fork this repository and modify the e2e [example](https://github.com/pact-foundation/pact-go/blob/master/dsl/pact_integration_test.go)  to quickly recreate your issue.
+1. An executable code example where possible. You can fork this repository and modify the e2e [example](https://github.com/pact-foundation/pact-go/blob/master/dsl/pact_test.go)  to quickly recreate your issue.
```
Comment (mohanraj-r) [inline CONTRIBUTING.md]: Not sure if this is correct - `pact_test.go` was the closest I could find to the integration test.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -83,5 +83,5 @@ You can also:
 
 ```
 cd dsl
-PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_Integration
+PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_AddInteraction
```
Comment (mohanraj-r) [inline CONTRIBUTING.md]: Not sure if this is correct either - just the closest test I could find. But then I don't see any uses of the env vars `PACT_INTEGRATED_TESTS`, `PACT_BROKER_USERNAME` or `PACT_BROKER_PASSWORD` in `dsl` dir

These env vars don't seem to be making a diff
```bash
# with env vars
$ PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_AddInteraction
PASS
ok      github.com/pact-foundation/pact-go/dsl  10.039s

# without env vars
$ go test -run TestPact_AddInteraction
PASS
ok      github.com/pact-foundation/pact-go/dsl  10.037s
$
```

Comment (mefellows) [reply CONTRIBUTING.md]: I would just remove steps 1, 2 and 3 from the `Integration Tests` section, they are not longer valid.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mefellows) [reply CONTRIBUTING.md]: Good spot! Just point them at the `examples` folder - they are all E2E tests of sorts. Running `make pact` will execute these.

Comment (mohanraj-r) [reply CONTRIBUTING.md]: ok sounds good

File: CONTRIBUTING.md (hunk 2 above)
Comment (mohanraj-r) [reply CONTRIBUTING.md]: so the section then would only mention about running `make pact` ?

**Comment by mohanraj-r** [comment]
@mefellows updated doc as requested

**Comment by mefellows** [comment]
Thanks!

**Pull request merged**